ow_faults/include/ow_faults/FaultInjector.h

```diff
@@ -39,14 +40,17 @@ class FaultInjector
   enum ComponentFaults {
     // general
     Hardware=1, 
+    //pt
+    JointLimit = 2,
     //arm 
-    TrajectoryGeneration=2, 
+    TrajectoryGeneration = 2,
     Collision=3, 
     Estop=4, 
     PositionLimit=5, 
     TorqueLimit=6, 
     VelocityLimit=7, 
-    NoForceData=8};
+    NoForceData=8
+    };
```

Comment:
Can you stay consistent in the spaces between the enum and the value? I think C++ community prefers adding a singe space before and after the equal sign.

Reply:
will do